src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/DomainEventService.kt

```diff
@@ -196,6 +200,17 @@ class DomainEventService(
       nomsNumber = domainEvent.data.eventDetails.personReference.noms,
     )
 
+  @Transactional
+  fun savePlacementApplicationWithdrawnEvent(domainEvent: DomainEvent<PlacementApplicationWithdrawnEnvelope>) =
+    saveAndEmit(
+      domainEvent = domainEvent,
+      typeName = "approved-premises.placement-application.withdrawn",
+      typeDescription = "An Approved Premises Request for Placement has been withdrawn",
```

Comment:
Again, it makes sense to refer to the placement application as a "request for placement"? Because we consider "Placement application" to be an entirely internal term which we don't wish to expose to other HMPPS services? Plus the fact that we are effectively in the process of renaming it?